components/Starknet/modules/ROOT/pages/index.adoc

```diff
@@ -1,7 +1,189 @@
 [id="overview"]
 = Overview
 
-Starknet is a permissionless, decentralized validity rollup, also known as a zero-knowledge rollup (ZK rollup). It operates as a Layer 2 (L2) network over Ethereum, enabling any dApp to achieve unlimited scale for its computation - without
-compromising Ethereum’s composability and security - thanks to Starknet’s reliance on the safest and most scalable cryptographic proof system: STARK.
+Starknet is a permissionless, decentralized zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2), enabling any dApp to achieve unlimited scale for its computation without compromising on Ethereum’s composability
+and security.
 
-Starknet contracts and the Starknet OS are written in Cairo – supporting the deployment and scaling of any use case, whatever the business logic.
+Starknet achieves low cost transactions and high performance by using its own cryptographic proof system called STARK. Starknet contracts and the Starknet OS are written in link:https://github.com/starkware-libs/cairo[Cairo], a custom-built and
+specialized programming language.
+
+== Explore Starknet
+
+[pass]
+++++
+<div class="home-cta-container">
+  <a href="https://docs.starknet.io/documentation/quick_start/environment_setup/" class="home-cta home-cta-first" id="cta1">
+    <div class="image-container">
+      <img src="_images/developers.svg" style="filter: none; border-radius: 0px;" class="cta-image">
+    </div>
+    <h2>Quick start</h2>
+    <p class="chakra-card__body css-jintet">Set up your environment and get started with Starknet.</p>
+    <p class="chakra-card__body css-jintet">Explore ></p>
```

Comment:
Can you make these bold or otherwise stand out?

Reply:
Add bolder and slightly larger font to highlight